proposals/NNNN-SendURI.md

```diff
@@ -0,0 +1,163 @@
+# SendURI
+
+* Proposal: [SDL-NNNN]((NNNN-SendURI.md))
+* Author: [Michael Crimando](https://github.com/MichaelCrimando) & [Derrick Portis](https://github.com/dportis)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core / iOS / Android / RPC ]
+
+## Introduction
+
+The SendURI feature is a new RPC that would allow SDL Apps on systems that contain built-in web browsers, mail services, SMS services and more to quickly and easily direct users to resource identifiers (URI).
+
+## Motivation
+
+In the future, vehicles may contain web browsers that a user could use (while stationary or in motion - depending on OEM), rear family entertainment systems that are SDL-enabled may start to exist, and SDL-enabled autonomous vehicles will become common. Getting this feature would enable an SDL conneciton to the built-in features for situations where it may be more comfortable to use in the vehicle displays. Examples include but are not limited to, emails, calls, SMS messages, video content, real estate listings, news articles, restuarant menus, charging station info, flight info, and shopping information. Sometimes a SDL app can emulate certain views using display layouts - but the system might not be able to show the desired amount of information.
+
+## Proposed solution
+
+The SendURI RPC would bring new functionality to SDL by giving app the ability to explore web addresses and more outside of SDL apps. This will lead to more versatile interactions with SDL.  
```

Comment:
Can you clarify "...giving app the ability to explore web addresses and more outside of SDL apps."?  Should this be "giving apps the ability to explore..." or "giving users the ability to explore..."?